### PR TITLE
161 document new viz

### PR DESCRIPTION
**Pull request by ryanSoley**
closes: #161 

---

## What
  * updates the library reference to remove `ui` module and add `viz` module
  * replace the dashboard page in the docs with a new visualizations one
  * add/update notebooks for each visualization in `notebooks/viz`
    * reenable notebook tests for viz notebooks

## How to Test
  * run the notebooks in `notebooks/viz` and validate the visualizations look good
  * build the docs locally and validate that everything in the library reference and the new visualization section looks good
> https://capitalone.github.io/rubicon-ml/contributing.html#documentation

### CODE REVIEW

**Review by shania-m — APPROVED** [review]
LGTM 🥇

File: docs/source/visualizations.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+.. _visualizations:
```
Comment (shania-m) [inline docs/source/visualizations.rst:1]: For the Dataframe Plot and Metric Correlation Plot I only see the inserted pngs, not the output of the .show() can you check on your end if you see it?

**Review by shania-m — CHANGES_REQUESTED** [review]

File: docs/source/visualizations.rst (hunk 1 above)
Comment (ryanSoley) [reply docs/source/visualizations.rst:1]: yeah thats intentional cause we cant actually run the dash apps on the docs server. they should all just say `Dash is running on http://127.0.0.1:8050/` and show the png

**Review by shania-m — APPROVED** [review]
LGTM

**Pull request merged**